From 1076d4c45108b1edad48fcac0f203c8c4e521b79 Mon Sep 17 00:00:00 2001
Message-Id: <1076d4c45108b1edad48fcac0f203c8c4e521b79.1551951770.git.liu.denton@gmail.com>
In-Reply-To: <cover.1551951770.git.liu.denton@gmail.com>
References: <cover.1551940635.git.liu.denton@gmail.com>
	<cover.1551951770.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Thu, 7 Mar 2019 01:28:22 -0800
Subject: [PATCH v3 1/4] merge-options.txt: correct typo
To: Git Mailing List <git@vger.kernel.org>
Cc: Phillip Wood <phillip.wood123@gmail.com>,
    Junio C Hamano <gitster@pobox.com>

The scissors line is included after the human commit message. Therefore,
saying "scissors will be prepended" was incorrect. Update to say
"appended" which is more correct.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---

Notes:
    Note that this can be squashed into 6f06b6aeef (merge: add scissors line on
    merge conflict, 2019-01-22) if it makes life easier.

 Documentation/merge-options.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/merge-options.txt b/Documentation/merge-options.txt
index c2a263ba74..6fff647409 100644
--- a/Documentation/merge-options.txt
+++ b/Documentation/merge-options.txt
@@ -31,8 +31,8 @@ set to `no` at the beginning of them.
 	This option determines how the merge message will be cleaned up
 	before commiting or being passed on. See linkgit:git-commit[1] for more
 	details. In addition, if the '<mode>' is given a value of `scissors`,
-	scissors will be prepended to MERGE_MSG before being passed on in the case
-	of a merge conflict.
+	scissors will be appended to MERGE_MSG before being passed on in the
+	case of a merge conflict.
 
 --ff::
 	When the merge resolves as a fast-forward, only update the branch
-- 
2.21.0.370.g4fdb13b891

